rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java

```diff
@@ -99,19 +101,39 @@ public static void main(String[] args) throws Exception {
     public static class EnrichmentFunction
             extends RichCoFlatMapFunction<TaxiRide, TaxiFare, RideAndFare> {
 
+        private ValueState<TaxiRide> startState;
```

Comment:
should be `transient`
`private transient`
pls see details:
https://stackoverflow.com/a/72558397